src/tests/system/tests/test_ipa_trusts.py

```diff
@@ -60,3 +62,132 @@ def test_ipa_trusts__lookup_group_without_sid(ipa: IPA, trusted: GenericADProvid
     status = ipa.sssctl.domain_status(trusted.domain, online=True)
     assert "online status: offline" not in status.stdout.lower(), "AD domain went offline!"
     assert "online status: online" in status.stdout.lower(), "AD domain was not online!"
+
+
+@pytest.mark.importance("medium")
+@pytest.mark.ticket(jira="RHEL-14752")
+@pytest.mark.topology(KnownTopology.IPATrustIPA)
+def test_ipa_trusts__ipa_master_lookup_trusted_user(ipa: IPA, trusted: IPA):
+    """
+    :title: Basic IPA-IPA Trust lookup on IPA server
+    :setup:
+        1. Restart SSSD and clear cache on IPA server
+    :steps:
+        1. Resolve trusted domain admin user
+    :expectedresults:
+        1. User is resolved
+    :customerscenario: True
+    """
+    ipa.sssd.restart(clean=True)
+
+    # Resolve user
+    username = trusted.admin_fqn
+    id_user = ipa.tools.id(username)
+
+    # Resolve group
+    groupname = trusted.fqn("admins")
+    getent_group = ipa.tools.getent.group(groupname)
+
+    assert id_user is not None, "Trusted admin user not found!"
+    assert id_user.user.name == username, "Username does not match!"
+
+    assert getent_group is not None, f"No group {groupname} found!"
+    assert getent_group.name == groupname, f"Group name does not match {groupname}!"
+
+
+@pytest.mark.importance("medium")
+@pytest.mark.ticket(jira="RHEL-14752")
+@pytest.mark.topology(KnownTopology.IPATrustIPA)
+def test_ipa_trusts__lookup_trusted_user(client: Client, ipa: IPA, trusted: IPA):
+    """
+    :title: Basic IPA-IPA Trust lookup on IPA client
+    :setup:
+        1. Restart SSSD and clear cache on IPA client
+    :steps:
+        1. Resolve trusted admin user
+        2. Resolve group "admins@trusteddomain"
+    :expectedresults:
+        1. User is resolved
+        2. Group is resolved
+    :customerscenario: True
+    """
+    client.sssd.restart(clean=True)
+
+    # Resolve user
+    username = trusted.admin_fqn
+    id_user = client.tools.id(username)
+
+    # Resolve group
+    groupname = trusted.fqn("admins")
+    getent_group = client.tools.getent.group(groupname)
+
+    assert id_user is not None, "Trusted admin user not found!"
+    assert id_user.user.name == username, "Username does not match!"
+
+    assert getent_group is not None, f"No group {groupname} found!"
+    assert getent_group.name == groupname, f"Group name does not match {groupname}!"
+
+
+@pytest.mark.importance("critical")
+@pytest.mark.topology(KnownTopologyGroup.AnyIPATrust)
+def test_ipa_trusts__authentication_with_default_settings(client: Client, trusted: GenericProvider):
```

Comment:
We recently changed the test name for ``authentication_with_default_settings ``. Let's change the name to ```test_ipa_trusts__user_login```. We are now assuming default settings.

Reply:
Renamed.